### ISSUE TITLE
make issue

### ISSUE THREAD

**Issue by hsw28** [wilsonlab/mwsoft64#7]
attempting to make from mwsoft64 using make command gives following output:

```
		MWSoft Building: Library
------------------------------------------------------
Building:LIB
make[1]: Entering directory '/home/hannah/Programming/mwsoft64/src'
gcc -DDATE="\"`/bin/date`\"" -g -O3 -Wall iolib.c -c -I../src -I/usr/X11R6/include 
iolib.c: In function ‘System’:
iolib.c:1167:12: error: storage size of ‘status’ isn’t known
 1167 | union wait status;
      |            ^~~~~~
iolib.c:1167:12: warning: unused variable ‘status’ [-Wunused-variable]
iolib.c: In function ‘checkArgs’:
iolib.c:1577:10: warning: variable ‘val’ set but not used [-Wunused-but-set-variable]
 1577 |   double val;
      |          ^~~
iolib.c: In function ‘BeginStandardHeader’:
iolib.c:641:5: warning: ignoring return value of ‘getcwd’, declared with attribute warn_unused_result [-Wunused-result]
  641 |     getcwd(pathname,MAXPATHLEN);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
iolib.c: In function ‘System’:
iolib.c:1212:1: warning: control reaches end of non-void function [-Wreturn-type]
 1212 | }
      | ^
make[1]: *** [Makefile:21: iolib.o] Error 1
make[1]: Leaving directory '/home/hannah/Programming/mwsoft64/src'
mv -f src/iolib.a build/lib; \
cd ..;
mv: cannot stat 'src/iolib.a': No such file or directory
make[1]: Entering directory '/home/hannah/Programming/mwsoft64/src/extract'
gcc -DDATE="\"`/bin/date`\"" -g -g3 -O3 -Wall spikeparms2.c -o spikeparms2 -I../../src -I/usr/X11R6/include -L/usr/X11R6/lib ../..//build/lib/iolib.a -lX11 -lm
gcc: error: ../..//build/lib/iolib.a: No such file or directory
make[1]: *** [Makefile:37: spikeparms2] Error 1
make[1]: Leaving directory '/home/hannah/Programming/mwsoft64/src/extract'
make: *** [Makefile:67: extract] Error 2

```

The definitions.mk file call for links to /usr/X11R6/include & /usr/X11R6/lib which are antiquated, so I have made a symbolic links on my computer as below

```
sudo ln -s /usr/lib/X11/ /usr/X11R6/lib

sudo ln -s /usr/include/X11/ /usr/X11R6/include

```

**Comment by imalsogreg** [comment]
The main issue is probably
```
iolib.c:1167:12: error: storage size of ‘status’ isn’t known
 1167 | union wait status;
```
PR incoming that changes the type of `status` to `int`

**Comment by imalsogreg** [comment]
closed by #8
